README.md

```diff
@@ -214,7 +214,14 @@ const bot = new Telegraf(token);
 bot.on("text", ctx => ctx.reply("Hello"));
 
 // Start webhook via launch method (preferred)
-bot.launch({ webhook: { domain: webhookDomain, port: port } });
+bot.launch({ 
+  webhook: {
+    domain: webhookDomain, // https://example.com
+    port: port, // Use server's port
+    hookPath: webhookPath, // If not passed, Telegraf will generate a secure path for you.
+    secretToken: randomAlphaNumericString // Use "crypto.randomBytes(64).toString('hex')" to generate cryptographically secure string; Optional but recommmended for more security. Also have to provide while setting the webhook url. see https://core.telegram.org/bots/api#setwebhook
```

Comment:
> Also have to provide while setting the webhook url

This is extraneous - while using `bot.launch()`, this is the only place you need to pass `secretToken`. You don't need to, and shouldn't call `setWebhook` as well.

Reply:
From that I mean when user going to specify webhook to telegram api using get method they have to specify secret_token as parameter in the url.

Reply:
`bot.launch()` calls `setWebhook` internally. You must NOT call `setWebhook` yourself.

https://github.com/telegraf/telegraf/blob/7907a7a79c468dbe7f15b6582a73aa57c643a923/src/telegraf.ts#L280-L287

Reply:
Ohh okay, I wasn't aware of this. Let me edit the comment.

Reply:
Done